apis/python/src/tiledbsoma/read_iters.py

```diff
@@ -60,55 +57,3 @@ def concat(self) -> RT:
         """
         arrow_tables = pa.concat_tables(TableReadIter(self.sr))
         return self._from_table(arrow_tables)
-
-
-class SparseCOOTensorReadIter(SparseTensorReadIterBase[pa.SparseCOOTensor]):
```

Comment:
this should not be deleted, as it is still very much needed.  Only CSC/CSR suffer from the memory-use-scale-with-domain-size issue

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/952#discussion_r1108907410

Reply:
Restored on most recent commit as discussed above